dev-setup/init-s3-bucket.sh

```diff
@@ -12,3 +12,5 @@ aws s3api --endpoint http://localhost:9878 create-bucket --bucket bpg
 aws s3api --endpoint http://localhost:9878 put-object --bucket bpg --key eventlog/
 aws s3api --endpoint http://localhost:9878 put-object --bucket bpg --key warehouse/
 kill ${fwd_pid} || echo "No forward pid to kill"
+
+kubectl -n local-ozone port-forward svc/s3g 9878:9878
```

Comment:
This is needed since local BPG instance assumes `localhost:9878` is reachable for Ozone

Reply:
good point. can you add it to the [port-forward.sh](https://github.com/apple/batch-processing-gateway/blob/main/dev-setup/port-forward.sh), and modify the doc of [this step](https://github.com/apple/batch-processing-gateway/blob/main/docs/GETTING_STARTED.md#port-forward-the-postgresql)?